src/compute/filter.rs

```diff
@@ -140,10 +162,10 @@ fn null_filter_simd<T: NativeType + Simd>(
 
```

Comment:
@jorgecarleitao shall we rename this function name to something that does not have SIMD in it? We don't really utilize SIMD here.